gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/auth/TestDelegationTokenBindingImpl.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.hadoop.fs.gcs.auth;
+
+import com.google.cloud.hadoop.util.AccessTokenProvider;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.Text;
+
+import java.io.IOException;
+
+/**
+ * A test delegation token binding implementation
+ */
+public class TestDelegationTokenBindingImpl extends AbstractDelegationTokenBinding {
+
+  public TestDelegationTokenBindingImpl() {
+    super(TestTokenIdentifierImpl.KIND);
+  }
+
+  @Override
+  public AccessTokenProvider deployUnbonded() throws IOException {
```

Comment:
I've noticed sometimes the term "bonded" is used, and other times "bound". Would it make sense to consolidate into one single term, just for clarity?

Reply:
Well, as I've stated before, this is modeled after the related support in the s3a connector, and my desire is to eventually submit a PR to hadoop for some common interfaces/base classes for this work. As such, I think it would be good to keep them as similar as possible for now, to make it easier to perform that refactoring across implementations later. At that point, we can/will consider renaming. Is that fair?